web-form.js

```diff
@@ -276,11 +313,11 @@ function fillCanvasForm() {
           document.getElementById(f.name).value = "";
         } else {
           if (datePart === 'day') {
-            document.getElementById(f.name).value = date.getDate();
+            document.getElementById(f.name).value = document.getElementById('web-form-day').value;
           } else if (datePart === 'month') {
-            document.getElementById(f.name).value = date.getMonth() + 1;
+            document.getElementById(f.name).value = document.getElementById('web-form-month').value;
           } else if (datePart === 'year') {
-            document.getElementById(f.name).value = date.getFullYear().toString().substr(-2);
+            document.getElementById(f.name).value = document.getElementById('web-form-year').value;
```

Comment:
should be 2 digits, like original
```suggestion
            document.getElementById(f.name).value = document.getElementById('web-form-year').value.substr(-2);
```